### PR TITLE
mdhtml: init at 0.2.2

### PR DESCRIPTION
**Pull request by tomkoid**
## Description of changes

mdhtml is a simple CLI Markdown to HTML converter with styling support.

https://codeberg.org/Tomkoid/mdhtml

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/text/mdhtml/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      sha256 = "sha256-893pqrrTftzKqPYZgukV/yx2gkukVZWDTgg7ufx1MsY=";
+    };
+
+  vendorHash = null;
+
+  meta = with lib; {
+    description = "Really simple CLI Markdown to HTML converter with styling support";
+    homepage = "https://codeberg.org/Tomkoid/mdhtml";
+    license = licenses.mit;
+    maintainers = with maintainers; [ tomkoid ];
```
Comment (h7x4) [inline pkgs/tools/text/mdhtml/default.nix:25]: Just a few more attributes

```suggestion
    changelog = "https://codeberg.org/Tomkoid/mdhtml/releases";
    maintainers = with maintainers; [ tomkoid ];
    mainProgram = "mdhtml";
```

**Review by h7x4 — COMMENTED** [review]
Cool stuff! Just a few nits

**Comment by h7x4** [comment]
Also, please rebase your commits to look like this:

```
maintainers: add tomkoid
mdhtml: init at 0.2.2
```

**Comment by Janik-Haag** [comment]
> I don't think it's obligatory just yet, but a lot of packages tend to go into the new by-name

You don't need to do this now, there will be a later pr that moves everything possible via a script so don't bother for this pr, when you do another init just put it into pkgs/by-name since you don't have to edit the all-packages.nix

**Comment by Janik-Haag** [comment]
Result of `nixpkgs-review pr 261671` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mdhtml</li>
  </ul>
</details>

**Review by Janik-Haag — APPROVED** [review]
LGTM

**Review by h7x4 — APPROVED** [review]

**Pull request merged**